docker-compose.yaml

```diff
@@ -3,8 +3,7 @@ volumes:
   relational-db-data:
 services:
   db:
-    platform: linux/x86_64
-    image: percona:5.7.39
+    image: docker.io/mysql/mysql-server:8.0.32
```

Comment:
FWIW - We aren't using the percona image because as of 8.0.32 they don't have a multi-platform image available so Apple Silicon machines would get the Intel version which is 10-20x slower than they should be. The next version should be multi-platform, but I'll believe it when I see it.

Reply:
Thanks for the extra context!